pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBase.java

```diff
@@ -831,6 +831,9 @@ protected boolean enqueueMessageAndCheckBatchReceive(Message<T> message) {
     protected abstract void updateAutoScaleReceiverQueueHint();
 
     protected boolean hasEnoughMessagesForBatchReceive() {
+        if (batchReceivePolicy.getTimeoutMs() <= 0) {
```

Comment:
It seems like a break change if user set this to 0, then `batchReceive()` would return `Messages` with empty message immediately when there is no message to consume.

Reply:
@Jason918 fixed, use 1ms instead.